src/main/java/edu/illinois/library/cantaloupe/cache/S3MultipartAsyncOutputStream.java

```diff
@@ -229,7 +228,9 @@ public void run() {
     private boolean requestCreated;
 
     private String uploadID;
-    private int partIndex;
+    // Part number must start with 1
+    // minio will hang if this is 0
```

Comment:
@glenrobson just a comment. I think this is where AWS S3 specs v/s Minio's ones are different.

Reply:
I think AWS also requires parts to start at 1 but hopefully it wouldn't crash :-). https://docs.aws.amazon.com/AmazonS3/latest/API/API_UploadPart.html